src/modules/compliance/src/lib/ComplianceInterface.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "ComplianceInterface.hpp"
+
+// ---------------------------------------------------------------
+//                  MMI module C interface
+// ---------------------------------------------------------------
+
```

Comment:
Please stick with the comments style for this project. Here, for example use this:

// MMI module C interface

Including:

- Only one space after //
- No '----' or other separator lines.

Also note some words repeat here. MMI stands for "Management Module Interface"

You can simply say:

// Internal MMI implementation